src/sass/layout/header.scss

```diff
@@ -0,0 +1,33 @@
+header {
+    height: 100vh;
+}
+.nav {
+background-image: url("../image/bg-nav-header.jpg");
```

Comment:
wczytales zdjecie a nastepnie je rozciagnoles, nie jest to dobra praktyka poniewaz na duzych ekranach rozdzielczosc bd bardzo niska i tlo straci na jakosci